geomesa-spark/geomesa-spark-core/src/main/scala/org/locationtech/geomesa/spark/GeoMesaSparkKryoRegistrator.scala

```diff
@@ -87,6 +87,8 @@ object GeoMesaSparkKryoRegistrator {
 
   def register(sfts: Seq[SimpleFeatureType]): Unit = sfts.foreach(GeoMesaSparkKryoRegistrator.putTypeIfAbsent)
 
+  def register(sft: SimpleFeatureType): Unit = register(Seq(sft))
```

Comment:
Ok, just to be pedantic, I'd switch the implementations here;).

I think it is more natural for register(Seq) to delegate to register(sft).:)